JamGame/Assets/Scripts/TileBuilder/TileBuilderImpl.cs

```diff
@@ -55,12 +52,14 @@ public partial class TileBuilderImpl : MonoBehaviour
 
         public GameMode CurrentGameMode => validator.GameMode;
 
+        private GameObject stashRootObject;
         private Vector2Int stashPosition = new(-10, -10);
         private float boundingBoxMin = -100.0f;
         private float boundingBoxMax = 100.0f;
 
         private void Awake()
         {
+            stashRootObject = new GameObject("Stash Root Object");
```

Comment:
Better revert this approach, as it was more intuitive

Reply:
Previous approach requires a creating an empty GameObject in scene and link them. This approach does it automatically.

Reply:
This empty `GameObject` gives understanding about what's happening here, unlike the case where you generate in in runtime